examples/csharp/src/services/account.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Generic;
+using Service;
+
+public class Account : Service
+{
+    public Get()
+    {
+        paramsIn = new Dictionary<string, string>();
```

Comment:
Where are these instance variables (paramsIn and path) defined? I don't see them in Account, or in Service, and Service has no base class, so... ?